TestStack.BDDfy/Reporters/ReportModel.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TestStack.BDDfy.Reporters
+{
+    public class ReportModel
+    {
+        public List<ReportModel.Story> Stories { get; set; }
+
+        public ReportModel()
+        {
+            Stories = new List<Story>();
+        }
+
+        public class Story
```

Comment:
Just saw this. I still think it might be confusing for a new contributor to understand what's going on with name reuse.